newsite/package.json

```diff
@@ -12,7 +12,7 @@
     "bootstrap": "^4.3.1",
     "bootstrap-vue": "^2.0.0-rc.28",
     "core-js": "^2.6.5",
-    "jquery": "^1.9.1",
+    "jquery": "^3.4.0",
```

Comment:
The 1.x jquery dependency is a peer dep, check which one and verify compatibility before updating